test/FunctionalTests/HelloWorldTest.cs

```diff
@@ -9,68 +9,74 @@
 using Microsoft.AspNetCore.Testing.xunit;
 using Microsoft.Extensions.Logging;
 using Xunit;
+using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
 {
-    // Uses ports ranging 5061 - 5069.
     public class HelloWorldTests
     {
+        private readonly ITestOutputHelper _output;
+
+        public HelloWorldTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
         [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        //[InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:5061/", ServerType.Kestrel, ApplicationType.Portable)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:5062/", ServerType.Kestrel, ApplicationType.Portable)]
-        public Task HelloWorld_IISExpress(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, string applicationBaseUrl, ServerType delegateServer, ApplicationType applicationType)
+        //[InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, ServerType.Kestrel, ApplicationType.Portable)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ServerType.Kestrel, ApplicationType.Portable)]
```

Comment:
All of these tests have multiple `InlineData` attributes, even though all but one are commented out. I'd rather leave them as-is since there are issues to re-enable some of them.

However, the `ServerType.Kestrel` is totally vestigial, I'll clean that up.